python/rag/elastic-chatbot-rag-app/README.md

```diff
@@ -0,0 +1,263 @@
+# Elastic Chatbot RAG App
```

Comment:
No mention of Galileo in here. Can you update to include information on Galileo and how to set it up